auth/auth0/README.md

```diff
@@ -104,4 +104,69 @@ Note: The easy way to know if your `access_token` is a jwt, is to try to decode
 
 #### Can I use other auth0 clients? (native, spa, non interactive clients)
 
-Of course! Just be sure to have a correct `access_token` as auth0 response :wink:
\ No newline at end of file
+Of course! Just be sure to have a correct `access_token` as auth0 response :wink:
+
+#### How can I retrieve user data from auth0
+
+You have many user data available into auth0 (depends of which provider you choose).
+
+First of all, you need to set your scope as `openid email profile`.
+
+With `profile` in the scope, you will receive many more data (cf https://auth0.com/docs/user-profile/normalized/auth0)
```

Comment:
Some wording issues in 111 (_many_ user data.... _into_ auth0) and 115 (_many_ more data)